LEGO1/mxstillpresenter.cpp

```diff
@@ -0,0 +1,7 @@
+#include "mxstillpresenter.h"
+
+// OFFSET: LEGO1 0x10043430 STUB
```

Comment:
This is not the constructor of `MxStillPresenter`, but of a subclass of `MxStillPresenter`.
`MxStillPresenter`'s vtable is at 0x100d7a38
At 0x10043430, there is a sconstructor with vtable at 0x100d7ac8

Reply:
0x100D7AC8 is the vftable for `LegoMeterPresenter`. Evidence:
0x10043430 default ctor of the type. Only called by LegoObjectFactory::Create.
0x1000A15E compares the requested type argument against LegoObjectFactory.m_7C_MxAtomId_LegoMeterPresenter
0x10006FC8 (LegoObjectFactory ctor) grabs the pointer to m_7c and store is in local var `p`.
10007A0A-10007A26 `*p = MxAtomId("LegoMeterPresenter", (LookupMode)0);`
Elementary. :) *phew*

Reply:
I completely agree with this :)